pkg/config/config.go

```diff
@@ -32,19 +32,16 @@ type Config struct {
 	Tick          int
 	StartClock    string
 	MetricsTick   int
-	MetricsFile   []MetricsFileConfig
-	MetricsStdout MetricsStdoutConfig
+	MetricsLogger []MetricsLoggerConfig
 	Cluster       []NodeConfig
 }
 
 // Made public to be parsed from YAML.
 
-type MetricsFileConfig struct {
-	Path      string
-	Formatter string
-}
-
-type MetricsStdoutConfig struct {
+type MetricsLoggerConfig struct {
+	// Path is a file path in which the metrics is written.
+	Path string
```

Comment:
It is not necessarily a good idea, but we could accept `stdout` as well as `/dev/stdout` (and `stderr`) for convenience as the output destination.
If we take this way, the name `Path` should be changed, since `stdout` is not a path.